### ISSUE TITLE
OTP 21 build failure

### ISSUE THREAD

**Issue by ilovezfs** [klacke/yaws#338]
HEAD @ dd91bbb6b0b4de2e1deee95297ea92b88fec243f

```
==> make install
Making install in src
  GEN      ../ebin/yaws.app
  GEN      ../ebin/yaws.appup
  ERLC     ../ebin/yaws.beam
  ERLC     ../ebin/yaws_app.beam
  ERLC     ../ebin/yaws_ticker.beam
  ERLC     ../ebin/yaws_config.beam
  ERLC     ../ebin/yaws_server.beam
compile: warnings being treated as errors
yaws_server.erl:1046: ssl:ssl_accept/2: deprecated; use ssl:handshake/2 instead
yaws_server.erl:2155: erlang:get_stacktrace/0: deprecated; use the new try/catch syntax for retrieving the stack backtrace
yaws_server.erl:3403: erlang:get_stacktrace/0: deprecated; use the new try/catch syntax for retrieving the stack backtrace
yaws_server.erl:3412: erlang:get_stacktrace/0: deprecated; use the new try/catch syntax for retrieving the stack backtrace
make[1]: *** [../ebin/yaws_server.beam] Error 1
make: *** [install-recursive] Error 1
```
full log https://gist.github.com/ilovezfs/254c396a3682407004992e3db7a4737c

**Comment by vinoski** [comment]
Basically the same issue as #334 &mdash; work is ongoing to allow Yaws to continue to work with older Erlang/OTP releases and also with version 21.

**Comment by vinoski** [comment]
Fixed at commit ba31d14.
